polaris/dataset/_dataset.py

```diff
@@ -191,8 +186,8 @@ def _compute_checksum(
 
         # If the Zarr archive exists, we hash its contents too.
         zarr_md5sum_manifest = None
-        if zarr_root_path is not None:
-            zarr_hash, zarr_md5sum_manifest = compute_zarr_checksum(zarr_root_path)
+        if self.zarr_root_path is not None:
```

Comment:
Can this use the `uses_zarr` property instead?

Reply:
Absolutely! Good catch!